R/multitables.R

```diff
@@ -26,6 +26,41 @@ setMethod("[[", c("MultitableCatalog", "numeric"), function (x, i, ...) {
     getEntity(x, i, Multitable, ...)
 })
 
+#' @rdname catalog-extract
+#' @export
+setMethod("[[<-", c("MultitableCatalog", "character", "missing", "formula"),
+          function (x, i, j, value) {
+              stopifnot(length(i) == 1)
```

Comment:
Done, I've left my changes to `formulaToQuery()` since it looked from the comments around it that was how it was supposed to work, although it's not used here anymore. I can remove them if you think we won't need them. Thoughts?